spec/support/fake_analytics.rb

```diff
@@ -84,7 +84,7 @@ def track_event(event, original_attributes = {})
       method_name = caller
         .grep(/analytics_events\.rb/)
         &.first
-        &.match(/:in `(?<method_name>[^']+)'/)
+        &.match(/:in '[\w\d]*#(?<method_name>[^']+)'/)
```

Comment:
The caller method returns a [different format in 3.4](https://docs.ruby-lang.org/en/3.4/NEWS_md.html#label-Compatibility+issues):
```diff
- "identity-idp/app/services/analytics_events.rb:4409:in `idv_phone_confirmation_otp_submitted'"
+ "identity-idp/app/services/analytics_events.rb:4409:in 'AnalyticsEvents#idv_phone_confirmation_otp_submitted'"
```